newsletter/anniversary-spanish.html

```diff
@@ -0,0 +1,128 @@
+            src="https://github.com/episphere/dashboard/blob/dev/siteManagerDashboard/assets/images/anniversary-spanish-graphic.png?raw=true"
+            alt="" width="700px" style="border:0;">
+        <!--[if (gte mso 9)|(IE)]>
+            <v:rect xmlns:v="urn:schemas-microsoft-com:vml" fill="true" stroke="false" style="width:600px;height:374px;">
+              <v:fill type="tile" src="https://user-images.githubusercontent.com/30497847/224184907-80e79218-ad99-4660-bbb0-8a23a1f5dddc.png" />
+                <v:textbox inset="0,0,0,0">
+          <![endif]-->
+      </td>
+    <tr>
+
+    <tr width="100%">
+      <td style="padding: 0px 0 10px 30px;">
+        <p
+          style="margin:0;height: 75px; width: 760px; color: #FFFFFF; font-family: sans-serif; font-size: 18px;  letter-spacing: 0; line-height: 24px; text-align: justify;">
+          Durante el último año, sus aportes a Connect nos han ayudado a comenzar a desarrollar un recurso de investigación sobre cómo prevenir el cáncer. Su participación continua nos ayudará a crecer y utilizar dicho recurso para explorar las causas del cáncer y descubrir la mejor manera de prevenirlo. <br/><br/> <b>Agradecemos su compromiso en lo que respecta a su colaboración con nosotros para el futuro de la prevención del cáncer y lo alentamos a que 
+            <a href="https://cancer.gov/connect-prevention-study/espanol" style="color: #FCC200;">corra la voz</a>.</b> ¡Esperamos que forme parte de la comunidad de Connect durante muchos años más!
+          <br />
+        </p>
+      </td>
+    </tr>
+    <tr>
+      <td style="padding: 0 1px 100px 40px;">
+        <p style="margin:0;"></p>
+      </td>
+    </tr>
+    <tr>
+      <td style="padding:0;">
+        <table width="100%" style="border-spacing:0;">
+          <tr>
+            <td style="padding: 0 32px 3px 15px;">
+              <p style="margin:0;box-sizing: border-box; height: 1px; width: 760px; border: 1px solid #FFFFFF; ;"></p>
+            </td>
+          </tr>
+        </table>
+      </td>
+    </tr>
+    <tr>
+      <td style="padding:0;background-color: #2A72A5; padding-bottom: 20px;">
+        <table width="100%" style="border-spacing:0;">
+          <tr>
+            <td style="height: 92px; width: 559px; color: #FFFFFF; font-family: sans-serif; font-size: 18px; letter-spacing: 0; 
+							line-height: 24px; text-align: center; padding: 10px 0 20px 20px;">
+              <p style="margin:0;">Estudio Connect para la Prevención del Cáncer</p>
+              <a href="https://www.cancer.gov/" style="color: white" target="_blank">Instituto Nacional del Cáncer</a> | <a
+                href="https://www.nih.gov" style="color: white" target="_blank">Institutos Nacionales de Salud</a><a> |
+              </a><a href="https://www.hhs.gov/" style="color: white" target="_blank">Departamento de Salud y Servicios Humanos de EE. UU.</a> | <a href="https://www.usa.gov/" style="color: white" target="_blank">USA.gov</a>
```

Comment:
```suggestion
                href="https://www.nih.gov" style="color: white" target="_blank">Institutos Nacionales de Salud</a> |
              <a href="https://www.hhs.gov/" style="color: white" target="_blank">Departamento de Salud y Servicios Humanos de EE. UU.</a> | <a href="https://www.usa.gov/" style="color: white" target="_blank">USA.gov</a>
```

That middle <a> tag probably isn't needed.